Automattic/woocommerce-payments
# Add PMME loader to blocks cart

Fixes #9107

#### Changes proposed in this Pull Request

This PR adds the skeleton loader to the ProductDetails component which is used on the Blocks Cart using a similar logic to #9166.

![Jul-30-2024 13-54-18](https://github.com/user-attachments/assets/c2952ba4-bb86-4d46-8b6a-52024169e662)


#### Testing instructions

1. Enable one or more BNPL methods
1. Add something the cart and go to the blocks cart page
2. Ensure there's a loader for the BNPL message and the size of its container is not changed after it finishes loading
1. Update the quantity of an item and ensure the same loader works as expected

-------------------


**Post merge**
